samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/ObjectDetectionConsoleApp/OnnxModelScorer.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML;
+
+namespace ObjectDetection
+{
+    class OnnxModelScorer
+    {
+        private readonly string imagesLocation;
+        private readonly string imagesFolder;
+        private readonly string modelLocation;
+        private readonly MLContext mlContext;
+
+        private IList<YoloBoundingBox> _boxes = new List<YoloBoundingBox>();
+        private readonly YoloWinMlParser _parser = new YoloWinMlParser();
+
+        public OnnxModelScorer(string imagesLocation, string imagesFolder, string modelLocation)
+        {
+            this.imagesLocation = imagesLocation;
+            this.imagesFolder = imagesFolder;
+            this.modelLocation = modelLocation;
+            mlContext = new MLContext();
+        }
+
+        public struct ImageNetSettings
+        {
+            public const int imageHeight = 416;
+            public const int imageWidth = 416;
+            public const bool channelsLast = true;
```

Comment:
Remove -- this is not used.

Reply:
Ok. I have removed ChannelsLast variable.